gcloud/credentials.py

```diff
@@ -72,8 +73,34 @@ def get_credentials():
     return client.GoogleCredentials.get_application_default()
 
 
+def get_for_service_account_json(private_key_path, scope=None):
```

Comment:
Iffy on the variable name (`private_key_path` makes me think I should be passing in *just* the .p12 key, not a credential) -- but this is a nit only.